conversions/Forwards/src/main/java/com/agentdid127/resourcepack/forwards/impl/PackMetaConverter.java

```diff
@@ -75,7 +87,7 @@ else if (version >= Util.getVersionProtocol(packConverter.getGson(), "1.7.2")
             JsonObject meta = json.getAsJsonObject("meta");
             if (meta == null)
                 meta = new JsonObject();
-            meta.addProperty("game_version", Util.getVersionFromProtocol(packConverter.getGson(), version));
+            meta.addProperty("game_version", Util.getVersionFromProtocol(packConverter.getGson(), versionInt));
```

Comment:
This is a bug. versionInt is a MCMeta version, not a Minecraft version. This should be replaced with `to`

Reply:
Oop